team/_posts/2024-05-27-Parani-Paras.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: member
+title: "Paras Parani"
+position: Student Research Assistant 
+nickname: paras
+handle: paras1242
+email: ppara014@fiu.edu
+twitter: 
+github: paras1242
+scholar: 
+orcid: 
+osf: 
+image: /assets/images/team/parani.jpeg
+cv: /assets/pdfs/Paras_Parani_Resume.pdf
+alum: false
+---
+
+Paras Parani is currently a Master’s student at the Knight Foundation School of Computing & Information Sciences (KFSCIS) at Florida International University. He is also working as a Student Research Assistant under Professor Fahad Saeed, where he researches Vision Transformers and Large Language Models (LLMs) and fine-tunes them for specific research applications
```

Comment:
He is also working as a Student Research Assistant with Professor Fahad Saeed @Prof-S, to develop transformer and Large Language Models (LLMs) for biomedical data.

Reply:
See how to make this (@Prof-S ) hyperlink in the .md file.